superset-frontend/src/dashboard/components/Header/index.jsx

```diff
@@ -823,6 +835,15 @@ const Header = () => {
           }
         `}
       />
+
+      <UnsavedChangesModal
+        title="Save changes to your dashboard?"
```

Comment:
We might need to localize these with the `t` function here. If this happens inside the component, it's still best for localization to be at the usage level.